docs/resources/mws_workspaces.md

```diff
@@ -183,6 +183,31 @@ output "databricks_token" {
 
 In order to create a [Databricks Workspace that leverages AWS PrivateLink](https://docs.databricks.com/administration-guide/cloud-configurations/aws/privatelink.html) please ensure that you have read and understood the [Enable Private Link](https://docs.databricks.com/administration-guide/cloud-configurations/aws/privatelink.html) documentation and then customise the example above with the relevant examples from [mws_vpc_endpoint](mws_vpc_endpoint.md), [mws_private_access_settings](mws_private_access_settings.md) and [mws_networks](mws_networks.md).
 
+## OEM Term of Service
+
+-> **Note** The following describes how to tag the customer OEM workspace and have the customer sign the terms of service via a click through workflow
```

Comment:
Is there a public OEM doc?

Reply:
The doc team haven't published the public OEM doc yet

Reply:
```suggestion
-> **Note** The following describes how to tag the customer OEM workspace and have the customer sign the terms of service via a click through workflow. Please contact your Databricks representative for further questions.
```